docs/user/getting-started.md

```diff
@@ -149,6 +148,87 @@ $ kubectl get pods -n flightctl
 [...]
 ```
 
+### flightctl in ACM
+
+Create a values.yaml file with the following content, replace
+
+ 1. flightctl.MY.DOMAIN with your base domain.
+ 2. openShiftApiUrl with your actual OpenShift API URL.
```

Comment:
We should add the `storageClassName` to this list.

Reply:
added